server/services/store/sqlstore/blocks.go

```diff
@@ -521,7 +521,8 @@ func (s *SQLStore) getBoardCount(db sq.BaseRunner) (int64, error) {
 	query := s.getQueryBuilder(db).
 		Select("COUNT(*) AS count").
 		From(s.tablePrefix + "boards").
-		Where(sq.Eq{"delete_at": 0})
+		Where(sq.Eq{"delete_at": 0}).
+		Where(sq.Eq{"is_template": false})
```

Comment:
@wuwinson We are currently counting templates in board count for telemetry, which is wrong. But this may change some telemetry for boards count when this is released or we count templates, which means we should count template cards.